### PR TITLE
Add C# examples to High-level multiplayer

### PR DESCRIPTION
**Pull request by jaros3**
I translated the small bits of GDScript examples of multiplayer into C#.

### CODE REVIEW

**Review by raulsntos — CHANGES_REQUESTED** [review]
Thanks for contributing to the .NET documentation.

File: tutorials/networking/high_level_multiplayer.rst [hunk 1]
```diff
@@ -132,12 +140,29 @@ and passed to the ``MultiplayerAPI``.
...
+
+    // Create client.
+    var peer = new ENetMultiplayerPeer();
+    peer.CreateClient(IPAddress, Port);
+    Multiplayer.MultiplayerPeer = peer;
+
+    // Create server.
+    var peer = new ENetMultiplayerPeer();
+    peer.CreateServer(Port, MaxClients);
+    Multiplayer.MultuplayerPeer = peer;
```
Comment (AThousandShips) [inline tutorials/networking/high_level_multiplayer.rst:153]:
```suggestion
    Multiplayer.MultiplayerPeer = peer;
```

File: tutorials/networking/high_level_multiplayer.rst [hunk 2]
```diff
@@ -111,16 +111,24 @@ which will override ``multiplayer`` for the node at that path and all of its des
...
 
     # By default, these expressions are interchangeable.
     multiplayer # Get the MultiplayerAPI object configured for this node.
     get_tree().get_multiplayer() # Get the default MultiplayerAPI object.
 
+ .. code-tab:: csharp
+
+    // By default, these expressions are interchangable.
+    Multiplayer  // Get the MultiplayerAPI object configured for this node.
+    GetTree().GetMultiplayer()  // Get the default MultiplayerAPI object.
```
[the remark this answers is not in the document]
Comment (jaros3) [reply tutorials/networking/high_level_multiplayer.rst:125]: I don't think semicolon applies here, as this is certainly incomplete statement. How about:

```
    MultiplayerApi api = Multiplayer; // Get the MultiplayerAPI object configured for this node.
    MultiplayerApi api = GetTree().GetMultiplayer(); // Get the default MultiplayerAPI object.
```

File: tutorials/networking/high_level_multiplayer.rst [hunk 3]
```diff
@@ -239,6 +296,23 @@ The function ``multiplayer.get_remote_sender_id()`` can be used to get the uniqu
         var sender_id = multiplayer.get_remote_sender_id()
         # Process the input and affect game logic.
 
+ .. code-tab:: csharp
+
+    public void OnSomeInput() // Connected to some input
```
Comment (AThousandShips) [inline tutorials/networking/high_level_multiplayer.rst:301]:
```suggestion
    public void OnSomeInput() // Connected to some input.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).

File: tutorials/networking/high_level_multiplayer.rst [hunk 4]
```diff
@@ -239,6 +296,23 @@ The function ``multiplayer.get_remote_sender_id()`` can be used to get the uniqu
...
+    {
+        RpcId(1, MethodName.TransferSomeInput); // Send the input only to the server
```
Comment (AThousandShips) [inline tutorials/networking/high_level_multiplayer.rst:303]:
```suggestion
        RpcId(1, MethodName.TransferSomeInput); // Send the input only to the server.
```

File: tutorials/networking/high_level_multiplayer.rst [hunk 5]
```diff
@@ -239,6 +296,23 @@ The function ``multiplayer.get_remote_sender_id()`` can be used to get the uniqu
...
+    }
+
+
+    // CallLocal is required if the server is also a player.
+    [Rpc(MultiplayerApi.RpcMode.AnyPeer, TransferMode = MultiplayerPeer.TransferModeEnum.Reliable, CallLocal = true)]
+    public void TransferSomeInput()
+    {
+        // The server knows who sent the input.
+        long senderId = Multiplayer.GetRemoteSenderId();
```
Comment (AThousandShips) [inline tutorials/networking/high_level_multiplayer.rst:312]: This is `int` in the API, not `long`

File: tutorials/networking/high_level_multiplayer.rst [hunk 6]
```diff
@@ -163,24 +188,35 @@ The rest are only emitted on clients:
...
+
 To check whether the peer is server or client:
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     multiplayer.is_server()
 
+.. code-tab:: csharp
```
Comment (AThousandShips) [inline tutorials/networking/high_level_multiplayer.rst:207]:
```suggestion
 .. code-tab:: csharp
```

**Pull request closed without merging**